_episodes/02-filedir.md

```diff
@@ -520,7 +520,7 @@ directory name to change our working directory.
 `cd` stands for 'change directory',
 which is a bit misleading:
 the command doesn't change the directory,
-it changes the shell's idea of what directory we are in.
+it changes the shell's idea of what directory we are in. The 'cd' command is akin to double clicking a folder in a graphical interface to get into a folder.
```

Comment:
Please split this line after the full stop.

Reply:
Done.